### DISCUSSION TITLE
How do i open GitHub Web Editor aka GitHub VScode?

### DISCUSSION THREAD

**Discussion by SangeethaPrabhu2021** [community/community#69014] [New to GitHub]
Is it possible to open GitHub Web Editor aka GitHub VScode?

**Comment by mixed-farming** [comment]
1) You can instantly open a VS Code web editor for any GitHub repository by pressing the period "." key while you are on that repository's page.
2) Alternatively, you can access the VS Code web editor for any GitHub repository by modifying the URL from "GitHub.com" to "GitHub.dev" in your web browser's address bar.

[For more information](https://www.youtube.com/watch?v=-sfsTh6od1k)

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
